ebpfsvc/api.cpp

```diff
@@ -6,6 +6,14 @@
 #include "pch.h"
 #include <stdio.h>
 #include "rpc_interface_h.h"
+#include "ebpf_windows.h"
+#include "Verifier.h"
+
+void
+cache_map_file_descriptors(const EbpfMapDescriptor* map_descriptors, uint32_t map_descriptors_count);
+
+void
+clear_map_descriptors();
```

Comment:
Unless these are static (i.e., assuming they're implemented in one file and called from another one), these should be in a .h file.

Reply:
Fixed.